src/libs/actions/Session/index.js

```diff
@@ -90,11 +90,31 @@ function signOutAndRedirectToSignIn() {
  * @param {String} [login]
  */
 function resendValidationLink(login = credentials.login) {
-    Onyx.merge(ONYXKEYS.ACCOUNT, {isLoading: true});
-    DeprecatedAPI.ResendValidateCode({email: login})
-        .finally(() => {
-            Onyx.merge(ONYXKEYS.ACCOUNT, {isLoading: false});
-        });
+    const optimisticData = [{
+        onyxMethod: CONST.ONYX.METHOD.MERGE,
+        key: ONYXKEYS.ACCOUNT,
+        value: {isLoading: true},
+        errors: null,
+        message: '',
```

Comment:
```suggestion
        value: {
            isLoading: true,
            errors: null,
            message: '',
        },
```

You put the 'message' and 'errors' outside of 'value'

Reply:
ohh crap nice catch!